HadesAPI/router.go

```
@@ -111,6 +189,21 @@ func addBuildToQueue(c *gin.Context, producer hades.JobPublisher) {
 		return
 	}
 
+	// Announce the job as Queued so lifecycle subscribers (HadesLogManager, the
+	// dashboard live feed) see it immediately. This is best-effort: a publish
+	// failure must not fail the enqueue that already succeeded.
+	if statusPublisher != nil {
+		if err := statusPublisher.PublishJobStatus(c.Request.Context(), buildstatus.StatusQueued, p.QueuePayload.ID.String()); err != nil {
```

golangci-lint (2.12.2)

[error] QF1008: could remove embedded field "QueuePayload" from selector

(staticcheck)